arches_querysets/querysets.py

```diff
@@ -152,6 +152,12 @@ def get_tiles(
                 raise ValueError(f"No nodes found for graph with slug: {graph_slug}")
 
         alias_expressions = generate_node_alias_expressions(self.model, nodes)
+
+        # arches_version==9.0.0
+        if arches_version < (8, 0):
+            msg = "arches-querysets requires all nodes to have an alias."
+            assert None not in alias_expressions, msg
```

Comment:
This is the better error for the issue we discussed in modular reports planning last week.